src/PrettierEditProvider.ts

```diff
@@ -22,6 +22,7 @@ import {
     ParserOption,
 } from './types.d';
 
+const bundledPrettier = require('prettier') as Prettier;
```

Comment:
I'm wondering what's the reason to not use es6 imports here and in src/configCacheHandler.ts? like `import prettier from 'prettier';`